pkgs/by-name/sc/screenly-cli/package.nix

```diff
@@ -0,0 +1,48 @@
+{ darwin
+, fetchFromGitHub
+, lib
+, perl
+, pkg-config
+, openssl
+, rustPlatform
+, stdenv
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "screenly-cli";
+  version = "0.2.3";
+
+  src = fetchFromGitHub {
+    owner = "screenly";
+    repo = "cli";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-rQK1EYb1xYtcxq0Oj4eY9PCFMoaYinr42W8NkG36ps0=";
+  };
+
+  cargoPatches = [
+    ./add-cargo-lock.patch
```

Comment:
Can we use `fetchpatch` with https://github.com/Screenly/cli/commit/898bd2e5e3a9653e3c3dde17e951469885734c40?

Reply:
Good idea, done!